ocs_ci/utility/utils.py

```diff
@@ -352,21 +352,42 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def mask_secrets(cmd, secrets):
```

Comment:
it's not cmd specific and can be used for whatever string, so can be called just like text_to_mask or just text. Update documentation accordingly please